openmeetings-service/pom.xml

```diff
@@ -43,5 +43,17 @@
 			<artifactId>wicket-core</artifactId>
 			<version>${wicket.version}</version>
 		</dependency>
+        <dependency>
+            <groupId>com.github.caldav4j</groupId>
+            <artifactId>caldav4j</artifactId>
+            <version>0.9-SNAPSHOT</version>
+			<exclusions>
```

Comment:
On caldav4j, it uses `net.sf.ehcache:ehcache:jar:2.10.2:compile` slightly different, but you end up with two libraries of ehcache. Since, it's redundant, hence the exclusion.